icu4c/source/data/buildtool/filtration_schema.json

```diff
@@ -53,6 +53,10 @@
             },
             "additionalProperties": false,
             "required": ["directory", "replacements"]
+        },
+        "collationUCAData": {
```

Comment:
userguide/icu_data/buildtool.md shows a category "coll_ucadata". Can/should this option be in/on that category?

Reply:
Discussed.